.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5502
+}
```

Comment:
I imagine you'll want to remove this file from the commit. It's just a settings file for your VSCode.